components/map/layers/bubble.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Bubble
+page_title: Map Layers - Bubble
+description: Discover the Blazor Map Bubble Layer and explore the examples.
+slug: components/map/layers/bubble
+tags: telerik,blazor,map,layers,bubble
+published: True
+position: 9
+---
+
+# Bubble Layer
+
+The **Bubble** type of layer lets you create circular shapes on a geographic position with a radius that is calculated via the value associated with the shape.
```

Comment:
In fact, there is the "Symbol.Square" type of bubble, so we the "circular" word can be removed. Something like "shapes of different type on a ..." can be used instead.